pkg/grpc/api.go

```diff
@@ -1509,6 +1517,130 @@ func (a *api) SetMetadata(ctx context.Context, in *runtimev1pb.SetMetadataReques
 	return &emptypb.Empty{}, nil
 }
 
+func (a *api) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	if in.WorkflowType == "" {
```

Comment:
It looks like we're not actually using `in.WorkflowType` for anything. I think we should either include it in the request or exclude this from the API. I believe neither Temporal nor the embedded engine will require this, so it might make sense to just remove it until we identify a component/scenario that requires it.